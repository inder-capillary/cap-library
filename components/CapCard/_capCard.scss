@import '../styles/_variables.scss';


.cap-card{
  width: 276px;
  height: 344px;
  max-height: 344px;
}

.ant-card-cover{
    padding-left: 20px;
    padding-right: 20px;
    background-color: #ECECEC;
}

.ant-card-head{
  padding: 0 12px;
}