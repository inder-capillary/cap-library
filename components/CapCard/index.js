/**
*
* CapCard
*
*/

import React from 'react';
import classNames from 'classnames';
import { Card } from 'antd';
import './_capCard.scss';

const clsPrefix = 'cap-card-v2';


function CapCard(props) {
  const { className, ...rest } = props;
  return (
    <Card className={classNames(clsPrefix, className)} {...rest} />
  );
}

CapCard.propTypes = {

};

export default CapCard;
