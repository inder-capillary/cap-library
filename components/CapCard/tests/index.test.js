import renderer from 'react-test-renderer';
import React from 'react'; // eslint-disable-line no-unused-vars
import CapCard from '../index';

describe('<CapCard />', () => {
  it('CapCard renders correctly', () => {
    const tree = renderer.create(<CapCard />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
