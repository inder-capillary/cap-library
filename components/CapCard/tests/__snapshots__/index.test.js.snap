// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CapCard /> CapCard renders correctly 1`] = `
<div
  className="ant-card cap-card ant-card-bordered"
>
  <div
    className="ant-card-body"
    style={Object {}}
  />
</div>
`;
